### PR TITLE
UI updates on pin entry screen 

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json]: don't we have a large logo already in the assets?

File: brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift [hunk 2]
```diff
@@ -173,16 +173,16 @@ class LoginViewController: UIViewController, Subscriber, Trackable {
         backgroundView.backgroundColor = Theme.primaryBackground
         pinViewContainer.constrain(toSuperviewEdges: nil)
         debugLabel.constrain([
-            debugLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: C.padding[3]),
+            debugLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: C.padding[6]),
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift]: you should set it to safeArea topAnchor

File: brd-ios/breadwallet/src/Views/PinView.swift [hunk 3]
```diff
@@ -18,7 +18,7 @@ class PinView: UIView {
 
     // MARK: - Public
     var itemSize: CGFloat {
-        return 24.0
+        return 22.0
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Views/PinView.swift:21]: why?

File: brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json (hunk 1 above)
Comment (dangelovska) [reply brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json]: no, we don't have it

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json]: LogoBlue is a vector and can be resized infinitely. you don't need a larger version of it :)

File: brd-ios/breadwallet/src/Views/PinView.swift (hunk 3 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/Views/PinView.swift:21]: smaller pin dots
Screenshot 2022-04-21 at 13 15 37

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Views/PinView.swift:21]: I hope this doesn't break anything as the pin pad is being used in many other places.

File: brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json (hunk 1 above)
Comment (dangelovska) [reply brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json]: okay then, updated :)

**Review by kenan-mamedoff — APPROVED** [review]

**Pull request merged**